concent_api/concent_api/system_check.py

```diff
@@ -93,6 +93,75 @@ def check_settings_storage_cluster_address(app_configs, **kwargs):  # pylint: di
     return []
 
 
+@register()
+def check_settings_storage_server_internal_address(app_configs, **kwargs):  # pylint: disable=unused-argument
```

Comment:
1) please add unit tests for this check
2) please extract error to a fixture/function, like `def create_error_13_ssl_cert_path_is_none():`